From: Andrew Morton <akpm@linux-foundation.org>
Subject: fs/ext4/super.c: remove unused `deprecated_msg'
Date: Fri Oct 21 08:05:49 AM PDT 2022

fs/ext4/super.c:1744:19: warning: 'deprecated_msg' defined but not used [-Wunused-const-variable=]

Reported-by: kernel test robot <lkp@intel.com>
Cc: Theodore Ts'o <tytso@mit.edu>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---


--- a/fs/ext4/super.c~fs-ext4-superc-remove-unused-deprecated_msg
+++ a/fs/ext4/super.c
@@ -1741,10 +1741,6 @@ static const struct fs_parameter_spec ex
 
 #define DEFAULT_JOURNAL_IOPRIO (IOPRIO_PRIO_VALUE(IOPRIO_CLASS_BE, 3))
 
-static const char deprecated_msg[] =
-	"Mount option \"%s\" will be removed by %s\n"
-	"Contact linux-ext4@vger.kernel.org if you think we should keep it.\n";
-
 #define MOPT_SET	0x0001
 #define MOPT_CLEAR	0x0002
 #define MOPT_NOSUPPORT	0x0004
_
